### PR TITLE
WAZO-3769 saml global config

### PR DESCRIPTION
**Pull request by rbienvenu0**

### CODE REVIEW

**Comment by rbienvenu0** [comment]
recheck

File: wazo_auth/services/saml.py [hunk 1]
```diff
@@ -57,26 +57,34 @@ def __init__(self, config: Config, tenant_service: TenantService):
...
+            value = global_saml_config.get(key)
+            if not value:
+                missing_keys.append(key)
+
+        if missing_keys:
+            logger.error(
+                'Failed to initialize SAML service, missing configuration %s',
+                ','.join(missing_keys),
+            )
+            return
```
Comment (DrPyser) [inline wazo_auth/services/saml.py:78]: Is it desirable to leave the SAMLService partially initialized?
SAML authentication attemps will likely result in errors when trying to access `self._saml_clients[domain]`, do we want to have better error handling there?

File: wazo_auth/config.py [hunk 2]
```diff
@@ -128,6 +128,7 @@
         'xmlsec_binary': '/usr/bin/xmlsec1',
         'key_file': '/var/lib/wazo-auth/saml/server.key',
         'cert_file': '/var/lib/wazo-auth/saml/server.crt',
+        'allow_unknown_attributes': True,
```
Comment (DrPyser) [inline wazo_auth/config.py:131]: Does the behavior change with this new option?

Comment (pc-m) [reply wazo_auth/config.py:131]: @sboily is this option necessary to get Google Workspace to work

**Review by DrPyser — DISMISSED** [review]
Behavior seems generally good and consistent - saml config change is reflected in saml behavior(could not test `allow_unknown_attributes` yet)

File: wazo_auth/config.py (hunk 2 above)
Comment (pc-m) [reply wazo_auth/config.py]: I removed this option as it is not required at this time

**Review by DrPyser — APPROVED** [review]

**Pull request merged**